machine/machine.go

```diff
@@ -69,6 +69,9 @@ func (e *EndpointStore) _updateMachines(
 			m.M.Host = ahost.Hostname
 			m.M.Active = true
 			m.M.Aws = e.config.GetAwsInfo(ahost.AwsMetadata)
+			if ahost.AwsMetadata != nil {
+				m.M.Region = ahost.AwsMetadata.Region
```

Comment:
I don't like this because it's AWS specific. The concept of region is generic. This should wait for the mdb data schema to be updated to include region as a generic field.